### ISSUE TITLE
HuggingFace request not yet approved

### ISSUE THREAD

**Issue by yazanatibm** [facebookresearch/llama#750]
Hi, sorry to create an issue for this.

Similar to https://github.com/facebookresearch/llama/issues/724

I have not yet been granted access after over a week.

The emails are identical.

**Comment by jspisak** [comment]
cc @osanseviero

**Comment by osanseviero** [comment]
As mentioned in https://github.com/facebookresearch/llama/issues/724#issuecomment-1698700377 , the approval system is handled by Meta, not by Hugging Face. 

* To confirm, did you fill both meta form and HF form?
* Were both with the same main email?
* Did you get an email from Meta providing you access to the Meta-format models? (not through HF)

**Comment by yazanatibm** [comment]
> To confirm, did you fill both meta form and HF form?

Yes.

> Were both with the same main email?

Yes.

> Did you get an email from Meta providing you access to the Meta-format models? (not through HF)

No, I did not.

However I just tried doing this again right now. This time I did immediately receive an email from Meta AI with the unique custom download link. Not sure what was happening last week and prior, as I did retry several times with the same information.

On Hugging Face website, this note is still showing in the meta-llama repos:

> Your request to access this repo has been successfully submitted, and is pending a review from the repo's authors.

Though I suspect the license terms acceptance will now be visible by the repo authors for approval, in which case I will close this issue.

Thank you for your help on this.
